internal/k8s/k8s.go

```diff
@@ -97,6 +98,23 @@ func (c *Client) CreateAPIKeySecret(ctx context.Context, apiKey string) error {
 	return nil
 }
 
+func (c *Client) PatchClusterRole(ctx context.Context, clusterRoleName string, clusterRolePatchBytes []byte) error {
+
+	updatedRole, err := c.clientSet.RbacV1().ClusterRoles().Patch(
+		context.TODO(),
```

Comment:
```suggestion
		ctx, 
```

You have a context in the parent function, how come you don't want to use it here?

Reply:
but i do! :D